metamist_infrastructure/driver.py

```diff
@@ -553,7 +667,7 @@ def _etl_function(
                 max_instance_count=1,  # Keep max instances to 1 to avoid racing conditions
                 min_instance_count=0,
                 available_memory='2Gi',
-                available_cpu=1,
+                available_cpu='1',
```

Comment:
Interesting this has to be a string and not an integer :-)

Reply:
A bit bizarre, I'm sure they're casting whatever value there is, but my local linting in VSCode was complaining: https://www.pulumi.com/registry/packages/gcp/api-docs/cloudfunctionsv2/function/#available_cpu_python